app/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -184,12 +185,10 @@ public IdentifierHolder saveParticipant(Study study, Set<Roles> callerRoles, Str
             if (isNotBlank(participant.getExternalId())) {
                 externalIdService.reserveExternalId(study, participant.getExternalId());    
             }
-            account = accountDao.signUp(study, participant, study.isEmailVerificationEnabled());
+            account = accountDao.constructAccount(study, participant.getEmail(), participant.getPassword());
```

Comment:
Possible to remove this and do new Account() or Account.create() or something instead?